while-job-hunting.md

```diff
@@ -0,0 +1,17 @@
+# While job hunting
+
+### You're in a strong position
+
+Software developers are in high demand, and the skillset that CodeClan arms you with combined with your previous life experience makes you a great hire for any tech firm.
+
+Don't undersell yourself and don't go for jobs that wouldn't value your skills. Your skillset is worth more than an £18k FTE salary.
+
+### "This post requires two years experience..."
+
+Don't be put off by entry level jobs that ask for multiple years experience. It's a common tactic used in almost all industries in an attempt to deter unconfident applicants from applying. CodeClan graduates have entered posts that require 2-years experience fresh from the course. Employers will more often than not consider your previous career as contributing to your experience.
+
+### Utilise the recruiter network on LinkedIn
+
+Once you're in work and have 6 months experience under your belt, you'll learn to hate recruiters because they contact you every other day. But they can be really useful when you're first looking. Your previous working experience combined with your new technical skillset will make you stand out in their swaithe of Computer Science grads fresh out of university.
+
+Not only will they post jobs on LinkedIn, they'll also like posts made by other recruiters, so you'll get exposed to a large group of potential jobs. This spreadsheet is non-exhuastive but has a number of [recruiters worth connecting](https://docs.google.com/spreadsheets/d/1ifQVISMd_zQzuYV-MyHG4eS50QAdgDyo2p0sme60Rbc/edit?usp=sharing) with.
```

Comment:
I'm not sure if we want to make this actually public? This repo is entirely public and we may end up making this a static site at some point.

On the flipside - if we're ok with it being public - would it make sense as a list specifically in the survival guide here? Then we can PR/contribute without going through you @ShetlandJ?